packages/cli/src/commands/build/index.ts

```diff
@@ -106,6 +106,7 @@ export interface BuildsManifest {
   argv: string[];
   error?: any;
   builds?: SerializedBuilder[];
+  features?: { speedInsightsVersion: string; webAnalyticsVersion: string };
```

Comment:
@feugy FYI, this line is failing type checking: https://github.com/vercel/vercel/actions/runs/6421634817/job/17436437058?pr=10667#step:9:57

Fixed in my PR: https://github.com/vercel/vercel/pull/10667/files#diff-452a115a951a23488dd3257a83934cc9439918fbd41143a0ded219164dd13c6f